juju/model.py

```diff
@@ -1072,8 +1072,6 @@ def _get_series(self, entity_url, entity):
                     application_name = entity['Meta']['charm-metadata']['Name']
                 if not series:
                     series = self._get_series(entity_url, entity)
-                if not channel:
-                    channel = 'stable'
```

Comment:
Sorry to hijack this @johnsca, but while you're in here, would you please fix the docstring on line 1005 s/development/edge?